configuration/pih/htmlforms/section-mch-referral.xml

```diff
@@ -207,7 +207,7 @@
                         </obsgroup>
                     </tr>
                 </includeIf>
-                <includeIf velocityTest="$fn.getObs($encounter, 'PIH:12747').valueCoded.equals($fn.getConcept('PIH:YES'))">
+                <includeIf velocityTest="$fn.getObs($encounter, 'PIH:12747').valueCoded.equals($fn.getConcept('PIH:YES')) || $encounter.encounterType.uuid != '0CF4717A-479F-4349-AE6F-8602E2AA41D3'">
```

Comment:
Hey @jmbabazi  we actually don't need the encounterType checks is the referral form... it's set up so that a row only shows up if that referral has been selected on the form.  (So if it's not possible to select that referral type on a form, the row will never show up)